starfish/imagestack/imagestack.py

```diff
@@ -290,6 +292,43 @@ def xarray(self) -> xr.DataArray:
         """Retrieves the image data as an xarray.DataArray"""
         return self._data
 
+    def sel(self, indexers: Mapping[Indices, Union[int, tuple]]=None):
+        """Given a dictionary mapping the index name to either a value or a slice range, return an
+        Imagestack with each dimension indexed accordingly
+
+        Parameters
+        ----------
+        indexers : Dict[Indices, (int/tuple)]
```

Comment:
is (int/tuple) valid mypy? If so, I love that.

Reply:
I think I found somewhere that's official sphinx format...let me try and find it